scripts/publish/_publish_tools.sh

```diff
@@ -24,6 +24,41 @@ function publish_options {
   done
 }
 
+# Finds the version of the crate based on current working directory
+function crate_version_prefix {
+  local REPO_ROOT
+  REPO_ROOT=$(git rev-parse --show-toplevel)
+  "${REPO_ROOT}"/scripts/tools/cargo-dig.rs -v | sed -e 's/\([0-9]*\)\.\([0-9]*\)\.\([0-9]*\)/\1_\2_\3/'
+}
+
+function sanity_check_sys_crate {
```

Comment:
Might be worth validating that this information is correct too. Feel free to either include it in this PR or as a follow-up.

```
[package.metadata.aws-lc-sys]
commit-hash = "c2b14dfb82e1a4d0f2260e05cf07c3a864e52d99"
```